src/Dock.Avalonia/Controls/DockPanelSplitter.cs

```diff
@@ -65,11 +65,51 @@ protected override void OnDragDelta(VectorEventArgs e)
             }
         }
 
+        private DockPanel _panel;
+        private Size _previousParentSize;
+
         /// <inheritdoc/>
         protected override void OnAttachedToVisualTree(VisualTreeAttachmentEventArgs e)
         {
             base.OnAttachedToVisualTree(e);
 
+            _panel = Parent as DockPanel;
+
+            _previousParentSize = _panel.Bounds.Size;
```

Comment:
Here Bounds are `{0, 0, 0, 0}` so this can result in incorrect calculation in LayoutUpdated handler.